src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const lines = table.getElementsByTagName('tr');
+const tableBody = table.querySelector('tbody');
+const counter = lines[0].getElementsByTagName('td');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+function buttonChecker() {
+  if (counter.length === 10) {
+    appendColumn.disabled = true;
+  } else {
+    appendColumn.disabled = false;
+  }
+
+  if (counter.length <= 2) {
```

Comment:
The condition `counter.length <= 2` also relies on the static `counter` variable. This should be dynamically calculated to ensure accurate button disabling.